CMakeLists.txt

```diff
@@ -122,7 +122,13 @@ if (USE_SYSTEM_SUITESPARSE)
         endif()
     endif()
 
-    add_library(klusolve SHARED ${KLUSOLVE_SRC} src/klusolvex.def)
+    # add_library(klusolve SHARED ${KLUSOLVE_SRC} src/klusolvex.def)
+    if (DEFINED ENV{KLU_AS_STATIC})
+        add_library(klusolve STATIC ${KLUSOLVE_SRC} src/klusolvex.def)
+    else()
+        add_library(klusolve SHARED ${KLUSOLVE_SRC} src/klusolvex.def)
+    endif()
```

Comment:
I think it would be more consistent to use CMake's system to handle this, since it would be presented in the UI. Something like...

```cmake
SET(KLUSOLVE_LIB_TYPE "SHARED" CACHE STRING "Library type (SHARED or STATIC)")
SET_PROPERTY(CACHE KLUSOLVE_LIB_TYPE PROPERTY STRINGS SHARED STATIC)
```

and later no "if" would be required

```cmake
add_library(klusolve ${KLUSOLVE_LIB_TYPE} ${KLU_OBJS} ${SUITESPARSE_OBJS} ${SUITESPARSE_SRC} ${KLUSOLVE_SRC} src/klusolvex.def)
```